cdk/customResources/connectorPlugin/index.py

```diff
@@ -0,0 +1,65 @@
+#
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+#
+#
+
+import boto3
+import time
+
+client = boto3.client('kafkaconnect')
+
+
+def on_event(event, context):
+    print(event)
+    request_type = event['RequestType']
+    if request_type == 'Create': return on_create(event)
+    if request_type == 'Update': return on_update(event)
+    if request_type == 'Delete': return on_delete(event)
+
+    return
```

Comment:
the paranoid engineer in me would prefer to throw an error (or at least log) if an unhandled request type showed up here.

Reply:
+1

switch/case/default :)

Reply:
let me check if lambda has python 3.10 runtime available. Python was a bit late to the switch case party.